gems/aws-sigv4/lib/aws-sigv4/signer.rb

```diff
@@ -48,7 +48,7 @@ module SigV4
     # If you are using the AWS SDK for Ruby, you can use any of the credential
     # provider classes:
     #
-    # * AWS::SDK::Core::AssumeRoleCredentialProvider
+    # * AWS::SDK::Core::AssumeRoleCredentialsProvider
     # * AWS::SDK::Core::AssumeRoleWebIdentityCredentialProvider
```

Comment:
All of these docs need to be updated

Reply:
yeah - I started updating before realizing how much ins sigv4 gem needs to change.    Will make updates to it as part of a separate PR.